index.js

```diff
@@ -180,72 +180,89 @@
   //  Value :: Location
   var Value = 'Value';
 
-  //  _funcPath :: (Boolean, Array String, a) -> Nullable Function
-  function _funcPath(allowInheritedProps, path, _x) {
-    var x = _x;
-    for (var idx = 0; idx < path.length; idx += 1) {
-      var k = path[idx];
-      if (x == null || !(allowInheritedProps || has(k, x))) return null;
-      x = x[k];
-    }
-    return typeof x === 'function' ? x : null;
-  }
+  //  Prefix :: String
+  var Prefix = 'fantasy-land/';
```

Comment:
I would prefer a lower-case "p" here, since this is a member of `String` rather than a data constructor (`Constructor` and `Value` are intended to be viewed as the data constructors of the `Location` type).